tsv_file_parser.conf

```diff
@@ -0,0 +1,18 @@
+input{
+    file{
+        path => 'FILEPATH'
+		type => 'INDEX_TYPE'
+        start_position => 'beginning'
+        sincedb_path => '/dev/null'
+    }}
+filter {
+	grok {
+    match => { 'message' => '(%{DATA:ID}	%{BASE16FLOAT:AFR}	%{BASE16FLOAT:AMR}	%{BASE16FLOAT:EAS}	%{BASE16FLOAT:EUR}	%{BASE16FLOAT:SAS}	)' }
+  }}
+
+output{ 
+  elasticsearch {
+    codec => json
+    hosts => ['ES_COORDINATOR','ES_MASTER']  
```

Comment:
**Issue**: Hardcoded Elasticsearch host names ('ES_COORDINATOR', 'ES_MASTER') can lead to configuration errors and are not flexible for different deployment environments. <br> **Fix**: Use environment variables or external configuration settings to define Elasticsearch hosts. <br> **Code Suggestion**: 
 ```
 -    hosts => ['ES_COORDINATOR','ES_MASTER']
 +    hosts => ['${ES_HOST1}','${ES_HOST2}']
 ```